scripts/start_fabric.sh

```diff
@@ -83,7 +83,10 @@ fi
 
 # If already fabric network up and running skip start again.
 if [[ ! $(docker ps --format '{{.Names}}' |grep "peer*") ]]; then
+    # Start the fabric containers peers, orderers, cas
     minifab up -i $FABRIC_VERSION
+    # Create network profile file for avalon to use
+    minifab profilegen
```

Comment:
Please add a comment which file is created by execution of this command ?

Reply:
yes

Reply:
Is there an option to pass generated file name with minifab profilegen ? If not, please add a comment which file this command will create